x-pack/plugins/observability/public/pages/alerts/components/render_cell_value/render_cell_value.tsx

```diff
@@ -38,6 +38,17 @@ export const getMappedNonEcsValue = ({
   return undefined;
 };
 
+const getRenderValue = (mappedNonEcsValue: any) => {
+  // can be updated when working on https://github.com/elastic/kibana/issues/140819
+  const value = Array.isArray(mappedNonEcsValue) ? mappedNonEcsValue[0] : mappedNonEcsValue;
```

Comment:
do we need to do that?
```suggestion
  const value = Array.isArray(mappedNonEcsValue) ? mappedNonEcsValue.join() : mappedNonEcsValue;
```